08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,40 @@
-/*Add CSS rules*/
\ No newline at end of file
+* {
+  font-family: Georgia;
+}
+body {
+  margin: 100px;
+}
+.delivery {
```

Comment:
not only should delivery be floated, but the items inside it should be as well. I think you meant for this to say .delivery > div.